impl/src/main/java/org/apache/myfaces/lifecycle/clientwindow/ClientConfig.java

```diff
@@ -198,7 +198,7 @@ public String getUserAgent(FacesContext facesContext)
     /**
      * Users can overload this method to define in which scenarios a request should result
      * in an 'intercepted' page with proper windowId detection. This can e.g. contain
-     * blacklisting some userAgents.
+     * blocklisting some userAgents.
```

Comment:
isn't blacklisting the correct word here?

Reply:
Both mean the same thing, but the connotations are different.  It relates to blacklisting/whitelisting, which can be viewed via a racial lens.

Reply:
Got it!

Reply:
"blocklisting" is not a word.   If you want to replace "blacklist", then pick a real word or phrase to do so.   Not everyone speaks English as a first language, and if you start picking words that don't appear in dictionaries, you will be doing far more excluding than including.

Reply:
Yes I would suggest rewording it rather than using a non-existent word.